docs/api-testing/composer.md

```diff
@@ -746,47 +738,37 @@ When you write the value of the config, for the static part of the endpoint, you
 This toggle switches between the Visual and Code views in the Composer. You can make calls and add assertions for testing your APIs, and insert variables wherever needed. You can use either, depending on which you're more comfortable with.
 
 #### Visual View
-Guides you through creating API tests using automated real-time suggestions via predictive text. No coding experience is required.<br/><img src={useBaseUrl('img/api-fortress/2021/01/visualView.png')} alt="Test Composer Visual view"/>
+Guides you through creating API tests using automated real-time suggestions via predictive text. No coding experience is required.<br/><img src={useBaseUrl('img/api-testing/visualView.png')} alt="Test Composer Visual view"/>
 
 #### Code View
-Enables you to write tests here from scratch, if you feel more comfortable working in code.<br/><img src={useBaseUrl('img/api-fortress/2021/01/codeView.png')} alt="Test Composer Code view"/>
+Enables you to write tests here from scratch, if you feel more comfortable working in code.<br/><img src={useBaseUrl('img/api-testing/codeView.png')} alt="Test Composer Code view"/>
 
 ### Add Component
 This button displays all available [assertion components](/api-testing/composer/assertion-components/), [I/O components](/api-testing/composer/io-components/), and [logical components](/api-testing/composer/logical-components/).<br/>
-<img src={useBaseUrl('img/api-fortress/2021/01/addComponent.png')} alt="Add Component"/>
+<img src={useBaseUrl('img/api-testing/add-component-nav.png')} alt="Add Component"/>
 
 If a component is not valid for the operation you are conducting, it will not be made available to help avoid mistakes. For instance, if you don’t add a `POST` first, you cannot add a `POST` Body or `POST` Param.
 
 :::note
 Sauce Labs free trials may not give you access to all available components.
 :::
 
-### Transform Component
-Transforms an existing component into another component of the same type.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/01/transformComponent.png')} alt="Add Component"/>
 
 ### Delete Component
 Deletes a selected component from the test while using Visual view.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/01/deleteComponent.png')} alt="Delete Component"/>
-
-### Invoke Snippet
-Allows you to use a previously created code snippet stored in [The Vault](/api-testing/vault).<br/>
-<img src={useBaseUrl('img/api-fortress/2021/01/invokeSnippet.png')} alt="Invoke Snippet"/>
+<img src={useBaseUrl('img/api-testing/deleteComponent.png')} alt="Delete Component"/>
 
-### Export Snippet
-Allows you to export a selected code snippet to the vault in order to be re-used later, or in another test.<br/>
-<img src={useBaseUrl('img/api-fortress/2021/01/exportSnippet.png')} alt="Export Snippet"/>
 
 ### Save Test
```

Comment:
@aliciakrebs the label now says only "Save" do you think we should change the title?